force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.js

```diff
@@ -0,0 +1,87 @@
+import { LightningElement } from 'lwc';
+const devFundWeight = 0.23;
+const processAutoWeight = 0.30;
+const userIntWeight = 0.25;
+const testDebugWeight = 0.22;
+
+export default class PlatformDevCertCalculator extends LightningElement {
+
+    devFundamentalsScore = 50;
+    processAutomationScore = 50;
+    userInterfaceScore = 50;
+    testingScore = 50;
+
+    certificationScore = 90;
+    numberOfQuestions = 60;
+
+    showResources = false;
+
+    currentHistoryId = 0;
+
+    attemptHistory = [
+        {Id: 1, Score: 50},
+        {Id: 2, Score: 68},
+        {Id: 3, Score: 70},
+        {Id: 4, Score: 90}
+    ]
+    
+    calculateScore(){
+        let devFundWeightScore = this.devFundamentalsScore * devFundWeight;
+        let processAutoWeightScore = this.processAutomationScore * processAutoWeight;
+        let userIntWeightScore = this.userInterfaceScore * userIntWeight;
+        let testDebugWeightScore = this.testingScore * testDebugWeight;
+
+        this.certificationScore = devFundWeightScore + processAutoWeightScore + userIntWeightScore + testDebugWeightScore;
+
+        this.showResourceIfFailed();
+        this.addAttemptHistory(this.certificationScore);
+        
+    }
+
+    handleChange(event){
+        console.log(event.target.value, event.target.name);
+        const inputName = event.target.name;
+        let value = Number(event.target.value);
```

Comment:
Add in a validation and error toast if the value is less than 0 or greater than 100